src/main/target/FF_FORTINIF4/target.h

```diff
@@ -91,7 +91,7 @@
 #define USE_MAX7456
 #define MAX7456_SPI_INSTANCE    SPI3
 #define MAX7456_SPI_CS_PIN      PB3
-#define MAX7456_SPI_CLK         (SPI_CLOCK_STANDARD) // 10MHz
+#define MAX7456_SPI_CLK         (SPI_CLOCK_STANDARD * 2) // 10MHz, 5MHz normal, 7.14 MHz when OC'd with the *2 multiplier
```

Comment:
Did you have an issue caused by SPI clock being too fast with these targets (`FF_FORTINIF4`, `KAKUTEF4` and `OMNIBUSF4`) ?

A similar issue was addressed in #4150.

Reply:
Yes. A few of the test crew reported the OSD being flakey on those.

Reply:
Then `max7456_clock = HALF` should be used.